### PR TITLE
Add org.gnome.Games

### PR DESCRIPTION
**Pull request by Kekun**

### CODE REVIEW

File: org.gnome.Games.json [hunk 1]
```diff
@@ -0,0 +1,147 @@
...
+           We build them first, because some of them are very big, and they
+           change more rarely than the rest of the modules. */
+        "libretro-cores/libretro-bsnes.json",
+        "libretro-cores/libretro-gambatte.json",
+        "libretro-cores/libretro-mednafen_ngp.json",
+        "libretro-cores/libretro-mednafen_pce_fast.json",
+        "libretro-cores/libretro-nestopia.json",
+        "libretro-cores/libretro-pcsx_rearmed.json",
+        /* End libretro cores */
+        "org.lua.modules.json",
```
Comment (TingPing) [inline org.gnome.Games.json]: Can use https://github.com/flathub/shared-modules/tree/master/lua5.3 to reduce duplication.

File: org.gnome.Games.json [hunk 2]
```diff
@@ -0,0 +1,147 @@
+{
+    "app-id": "org.gnome.Games",
+    "runtime": "org.gnome.Platform",
+    "runtime-version": "3.24",
+    "branch": "stable",
+    "sdk": "org.gnome.Sdk",
+    "command": "gnome-games",
+    "tags": ["nightly"],
```
Comment (TingPing) [inline org.gnome.Games.json]: Wrong tag.

File: org.gnome.Games.json [hunk 3]
```diff
@@ -0,0 +1,147 @@
+{
+    "app-id": "org.gnome.Games",
+    "runtime": "org.gnome.Platform",
+    "runtime-version": "3.24",
+    "branch": "stable",
```
Comment (TingPing) [inline org.gnome.Games.json]: Can remove this, its set automatically.

**Comment by Kekun** [comment]
Thanks, all of this has been updated.

**Comment by TingPing** [comment]
As mentioned in person will need to do some testing on how this will handle the extension points and how they will work with appdata.

**Comment by bochecha** [comment]
> As  mentioned in person will need to do some testing on how this will  handle the extension points and how they will work with appdata.

Can't we push this to Flathub anyway?

The app works just fine without extensions, so why not push it directly, and figure out the extensions later? 😃 

(and FWIW, I'm currently hosting the PicoDrive extension on my own personal repo, and it works fine, just not the appdata)

**Comment by TingPing** [comment]
Test build: https://flathub.org/builds/#/builders/1/builds/168

File: org.gnome.Games.json [hunk 4]
```diff
@@ -0,0 +1,145 @@
...
+                }
+            ]
+        },
+        {
+            "name": "gnome-games",
+            "post-install": ["mkdir -p /app/lib/libretro/extra.d"],
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://git.gnome.org/browse/gnome-games/snapshot/gnome-games-0b68b134ec201e4a93829d772c37522a3b252a81.tar.xz",
```
Comment (TingPing) [inline org.gnome.Games.json]: If possible we do really want stable tagged releases.

**Comment by TingPing** [comment]
BTW this was still waiting on the build being fixed, currently the shared-modules submodule was never added.

**Comment by TingPing** [comment]
Test build: https://flathub.org/builds/#/builders/1/builds/253

**Comment by Kekun** [comment]
It failed: the included .json were using toplevel-relative paths, which I just fixed. Let's try that again. :)

**Comment by TingPing** [comment]
https://flathub.org/builds/#/builders/1/builds/274

**Comment by Barthalion** [comment]
PCSX-R core fails to build on ARMs as seen [here](https://flathub.org/builds/#/builders/1/builds/380).

**Comment by alexlarsson** [comment]
bot, build org.gnome.Games

**Comment by alexlarsson** [comment]
bot, build org.gnome.Games

**Comment by alexlarsson** [comment]
bot, build org.gnome.Games

**Comment by alexlarsson** [comment]
bot, build org.gnome.Games

**Comment by alexlarsson** [comment]
bot, build org.gnome.Games

**Comment by alexlarsson** [comment]
Yay, now it worked: https://flathub.org/builds/#/builders/1/builds/786
Sorry for spamming this PR with test.

**Comment by Kekun** [comment]
I just updated the branch to 3.26, hopefully it will build correctly this time.

**Comment by TingPing** [comment]
https://flathub.org/builds/#/builders/1/builds/790

**Comment by nedrichards** [comment]
New build: https://flathub.org/builds/#/builders/1/builds/794

**Comment by Kekun** [comment]
And it passed! o/

**Comment by nedrichards** [comment]
Just started reviewing :) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Comment by nedrichards** [comment]
Repository has been created: https://github.com/flathub/org.gnome.Games <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Pull request closed without merging**